target/linux/ramips/dts/mt7621_dlink_dir-xx60-a1.dtsi

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
```

Comment:
dts-v1 shouldn't be here: https://openwrt.org/docs/guide-developer/device-support-policies#dts-v1